client/ayon_core/lib/execute.py

```diff
@@ -209,6 +209,15 @@ def run_ayon_launcher_process(*args, **kwargs):
         # - fill more if you find more
         env = clean_envs_for_ayon_process(os.environ)
 
+    if add_sys_paths:
+        pp_set = frozenset(sys.path)
+        new_pythonpath = list(pp_set)
+        pythonpath = env.get("PYTHONPATH") or ""
+        for path in frozenset(pythonpath.split(os.pathsep)):
```

Comment:
This is wrong... the `frozenset` will lose ordering hence the paths you are appending will be in wrong order.